azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -142,6 +142,13 @@ def _format_results(self, sampler_seed=None):
 
     @staticmethod
     def _draw_random_sample(sampler_seed, probabilities, shots):
+        _norm = sum(probabilities.values())
+        if _norm != 1:

```

Comment:
Question, how common is that we have an exact value of 1 here to check for equality explicitly instead of letting the logic below with numpy.isclose() taka care of it even if we end up dividing by 1 in rare ocassions?

Reply:
I found that it's pretty rare for the sum to not be equal to 1 hence the if statement.